### DISCUSSION TITLE
suggestion: offer cheat sheets, especially when you override builtins

### DISCUSSION THREAD

**Discussion by colemickens** [tridactyl/tridactyl#4782] [Ideas]
I have tried to adopt Tridactyl many times and every time I hit the same issues.

I can't even figure out how to do the equivalent of Ctrl+F with tridactyl and it overrides the shortcut.

So even when I pull up the list of default keybindings *I can't even figure out how to search it*.

I also can't figure out how to even search open tabs and switch to one. Maybe I'm missing it, but I feel like there could be an easier on-ramp for folks.

**Comment by bovine3dom** [comment]
Have you looked at the `:tutorial`?

There's a PR opened for a cheat sheet but it got stale: https://github.com/tridactyl/tridactyl/pull/4355

It wouldn't have helped you in your case (to do it permanently you want  `:unbind <C-f>`, or you can use `<C-v><C-f>` to access it on a one-off basis)

**Reply by bovine3dom** [reply]
There's also this community-created cheat sheet here https://cheatography.com/goumlechat/cheat-sheets/tridactyl/ which does include the `<C-v>` command
